loader/src/main/scala/com/actian/spark_vector/loader/spark_sql_utils/SparkSqlTable.scala

```diff
@@ -0,0 +1,32 @@
+package com.actian.spark_vector.loader.spark_sql_utils
```

Comment:
This makes more sense to be in com.actian.spark_vector.sql

Also, in package names _ replaces -.